### DISCUSSION TITLE
[Weekly Admin Curation Report]: April 24 - 30

### DISCUSSION THREAD

**Discussion by AlexanderPico** [wikipathways/wikipathways-help#34] [curation-reports]
### Recent changes

Recent Changes: 108 pathways! To many to click through, so I used [this page](https://classic.wikipathways.org/index.php?title=Special:RecentPathwayChanges&limit=100&offset=0) instead.
 * Egon updated the old license info and made many pathway nodes clickable. Awesome! I did not bother reviewing each and every edit by Egon :)
 * Edits by other authors all looked good, including a tutorial pathway which was tagged and then deleted.

Under construction: 21 pathways. Lots of good stuff in here, just needed quick fixes:
 * Approved WP5123, WP5343, WP5344, WP5345, WP5348, WP5356
 * Removed color from WP5097, WP5347 and approved
 * Fixed titles and descriptions for WP5136, WP5324, WP5354 and approved
 * Added ontology tags and approved WP5346
 * Marked for deletion: WP5334, WP5335, WP5339, WP5341, WP5342
 * Still under construction: WP5336, WP5355, WP5360

### New pathways

List of new pathways: WP5097, WP5123, WP5136, WP5324

### Deleted Pathways

List of deleted pathways: WP5359, WP5214, WP1451, WP5308

### Update Approved pathway tag

- [X] All Approved pathways tags were updated!

**Comment by khanspers** [comment]
Thanks for the report Alex and thanks for editing the Curation Report template!

@DeniseSl22 : Some of the pathways tagged for deletion (WP5334, WP5335, WP5339, WP5341, WP5342) are from your students and the notes indicate the plan is to merge them. Please review.

**Reply by AlexanderPico** [reply]
Yeah, some of those looked good, but someone marked them as "tutorial" pathways, so...

@DeniseSl22 Please remove both tags, if not correct.

**Reply by DeniseSl22** [reply]
These pathways are actually from students from @tabbassidaloii , so she's in charge of the merging. I'm not sure who added the request deletion tags (at least I cannot remember that I did that)... I added the "test pathway", so that these pathways will not disturb the automated tests that are running.

**Reply by tabbassidaloii** [reply]
> These pathways are actually from students from @tabbassidaloii , so she's in charge of the merging. I'm not sure who added the request deletion tags (at least I cannot remember that I did that)... I added the "test pathway", so that these pathways will not disturb the automated tests that are running.

They are all merged (except for WP5342) here https://classic.wikipathways.org/index.php/Pathway:WP5356.
Is there a way to acknowledge the creator of the individual pathways in the merged one?

WP5342 needs some improvement/curation, so let's not yet approve it and tag it as a test pathway and under construction.

**Reply by DeniseSl22** [reply]
@tabbassidaloii : you cannot add the original creators using the classic website as an official author, unless they all individually make a small change to the pathway model (e.g. add their name to the description, one at a time?). You could also add their names in the description yourself, and later add them as authors in the new website?

**Reply by tabbassidaloii** [reply]
@DeniseSl22, Okay, thanks for the suggestions. I will add their names as you suggested.

**Reply by khanspers** [reply]
Thanks @tabbassidaloii and @DeniseSl22 ! So other than WP5342, can I go ahead and delete the other 4 pathways?

**Reply by tabbassidaloii** [reply]
Yes, those can be deleted. Thank you!
